process_manager/templates/process_manager/partials/process_table.html

```diff
@@ -1,2 +1,46 @@
+{% extends "django_tables2/table.html" %}
 {% load render_table from django_tables2 %}
+{% load querystring from django_tables2 %}
+{% block table.thead %}
+  <div>
+    <button style="padding: 5px 12px;
+                   background-color: #f8f9fa;
+                   color: #495057;
+                   border: 1px solid #ddd;
+                   border-radius: 8px;
+                   font-size: 0.9em;
+                   font-weight: 600;
+                   cursor: pointer;
+                   transition: background-color 0.3s ease"
+            onmouseover="this.style.backgroundColor='#e2e6ea'"
+            onmouseout="this.style.backgroundColor='#f8f9fa'"
+            hx-get="{% url 'process_manager:process_table' %}?sort="
+            hx-target="div.table-container"
+            onclick="document.getElementById('sort-field').value = ''; document.getElementById('sort-param-display').textContent = 'None';">
```

Comment:
I'm not entirely sure - will let @cc-a to comment - but I believe this can be re-written in hyperscript as:
```suggestion
            _="set #sort-field's value to '' and set #sort-param-display's textContent to 'None'">
```